src/test/test-stat-util.c

```diff
@@ -37,9 +37,15 @@ TEST(statx_definitions) {
         ASSERT_EQ(STATX_MNT_ID,         0x00001000U);
         ASSERT_EQ(STATX_DIOALIGN,       0x00002000U);
         ASSERT_EQ(STATX_MNT_ID_UNIQUE,  0x00004000U);
+#ifdef STATX_SUBVOL
         ASSERT_EQ(STATX_SUBVOL,         0x00008000U);
+#endif
```

Comment:
why doesn't linux/stat.h cover these? we carry our own copy of them after all

Reply:
musl's sys/stat.h conflicts with linux/stat.h, hence we cannot include linux/stat.h here.
glibc's sys/stat.h internally includes linux/stat.h, and our tree includes the latest linux/stat.h, hence all constants are available.